README.md

```diff
@@ -38,7 +38,7 @@ curl -L https://raw.githubusercontent.com/foundry-rs/starknet-foundry/master/scr
 You can also specify a version you wish to install:
 
 ```shell
-curl -L https://raw.githubusercontent.com/foundry-rs/starknet-foundry/master/scripts/install.sh | sh -s -- -v 0.1.0
+curl -L https://raw.githubusercontent.com/foundry-rs/starknet-foundry/master/scripts/install.sh | sh -s -- -v 0.2.0
```

Comment:
Let's bump to newest, `0.3.0`

Reply:
I feel like it could show that one can install a specific version, especially a past one.
The reason behind the bump is [here](https://github.com/foundry-rs/starknet-foundry/pull/407/files#r1284175257).

Reply:
Yes, this was fixed in 0.2.0
I suppose it's pretty self explanatory that any arbitrary version can be installed (since we are providing a version to the script) and we should probably always incliude latest version in docs, in case someone copy-pastes? (which I think can happen a lot)